test/Sentry.Unity.Editor.iOS.Tests/Sentry.Unity.Editor.iOS.Tests.csproj

```
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutDir>$(PackageEditorTestsPath)</OutDir>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="TestFiles/*.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../src/Sentry.Unity/Sentry.Unity.csproj">
+      <Private>false</Private>
+    </ProjectReference>
```

What about this format? To have it in a single line:

```suggestion
    <ProjectReference Include="../../src/Sentry.Unity/Sentry.Unity.csproj" Private="false" />
```